driver.js

```diff
@@ -12,12 +12,13 @@ console.log("Starting simulation.  This may take a moment...");
 // Creating genesis block
 let bc = Blockchain.createInstance({
   clients: [
-    {name: 'Alice', amount: 233},
-    {name: 'Bob', amount: 99},
-    {name: 'Charlie', amount: 67},
+    {name: 'Alice', amount: 233, password: 'alice_pswd'},
```

Comment:
It looks like you added a default for the password when it is not specified, which I like.  Let's leave it out of the driver file to keep it simple.  (We should probably add a mnemonic-driver.js to show how that portion could be used.)

Reply:
driver.js: removed 'password' field during client initialization. instead shifted these details in the new mnemonic-diver.js file